lib/seed/facility_seeder.rb

```diff
@@ -84,6 +88,7 @@ def call
           generating_seed_data: true,
           name: district_names[i],
           organization_id: organization.id,
+          protocol_id: protocol.id,
           state: nil)
       }
```

Comment:
While we're here, and if we have the time, can we refactor the call method so it's easier to read? Just extract functions where we have comments right now, even?

Reply:
I like this idea. I'll do this now.

Reply:
I just took a stab at this, and there's enough local temporary info (like `state_results`) that are being passed around that I think it's ok to leave as is. The alternative is passing these bits of info into the extracted methods, or saving them in state. Both seem a bit awkward to me.

What do you think?

Reply:
If it's not easy to do now with this PR we can leave it for later I guess. I don't mind passing around some variables, I could also take a stab at this separately.

Reply:
The changes look good @harimohanraj89!